### PR TITLE
Add MySQL example

### PR DESCRIPTION
**Pull request by hawkup**
This PR add MySQL example like the others example.

### CODE REVIEW

File: tests/mysql_test.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+export function teardown() {
+  db.close();
+}
+
+export default function () {
+  db.exec("INSERT INTO keyvalues (`key`, value) VALUES('plugin-name', 'k6-plugin-sql');");
+
+  let results = sql.query(db, "SELECT * FROM keyvalues WHERE `key` = ?;", 'plugin-name');
+  for (const row of results) {
+    console.log(`key: ${String.fromCharCode(...row.key.toString().split(','))}, value: ${String.fromCharCode(...row.value.toString().split(','))}`);
```
Comment (imiric) [inline tests/mysql_test.js:26]: Hhmm why is this `split()` and conversion required?

**Review by imiric — COMMENTED** [review]
Hey thanks for this!

File: tests/mysql_test.js (hunk 1 above)
Comment (hawkup) [reply tests/mysql_test.js:26]: Key and Name from results display as ASCII code. Then I convert it to String.

ASCII
CleanShot 2564-09-28 at 09 19 32@2x


After convert
CleanShot 2564-09-28 at 09 18 57@2x

Comment (imiric) [reply tests/mysql_test.js:26]: I understand, but I'm wondering why this is required for MySQL and not for the other RDBMSs.

We can merge this as is, but I'd like to clarify this, and I just don't have the time right now.

**Comment by imiric** [comment]
I'll merge this as is, since it would be helpful for others (see #10). And we can determine why it's returning as ASCII later.

Thanks again!

**Pull request merged**